anda/tools/flashrom-cros/flashrom-cros.spec

```diff
@@ -0,0 +1,37 @@
+%global commit  c1ab7468d28d164a30d598eb3e42a5febaf73bbc
+%global shortcommit %(c=%{commit}; echo ${c:0:7})
+
+Name:		flashrom-cros
+Version:	%shortcommit
+Release:	1%{?dist}
+Summary:	Simple program for reading/writing flash chips content (ChromiumOS fork)
+License:	GPL-2.0-only
+URL:		https://chromium.googlesource.com/chromiumos/third_party/flashrom
+Source0:	%url/+archive/refs/heads/release-R130-16033.B.tar.gz
+BuildRequires:	gcc gnupg2 libusb1-devel meson pciutils-devel python3-sphinx systemd zlib-devel dmidecode
+Requires:	dmidecode
+Requires:	udev
+ 
+ 
+%description
+flashrom is a utility for identifying, reading, writing, verifying and erasing
+flash chips. It is designed to flash BIOS/EFI/coreboot/firmware/optionROM
+images on mainboards, network/graphics/storage controller cards, and various
+other programmer devices.
+ 
+%prep
+%setup -C
+ 
+%build
+meson -Dtests=disabled -Ddocumentation=disabled -Duse_internal_dmi=false . build
+meson compile -C build
```

Comment:
can you use the `%meson -Dtests=disabled -Documentation=blah blah blah` and `%meson_build` macros instead?